app/clients/letter/dvla.py

```diff
@@ -6,6 +7,28 @@
 import requests
 from flask import current_app
 
+from app.clients import ClientException
+
+
+class DvlaException(ClientException):
+    pass
+
+
+class DvlaDuplicatePrintRequestException(DvlaException):
+    pass
+
+
+class DvlaUnauthorisedRequestException(DvlaException):
+    pass
+
+
+class DvlaRetryableExcepton(DvlaException):
```

Comment:
Should we also have a `DvlaNonRetryableException` (to mirror `EmailClientNonRetryableException`) which we could use to handle 400s from the API? Then we can handle deal with those separately in celery.

Also might be fine to leave this for now and deal with it on the celery task story.

Reply:
I was thinking of the `DvlaException` as the non-retryable one when I wrote this but can see that's not clear. I could maybe rename it to `DvlaNonRetryableException` and then just make the other errors inherit from `ClientException` directly?

Or the other option would be to keep the `DvlaException` (which all the errors inherit from but which is never used directly) and to also add a `DvlaNonRetryableException` exception type

Reply:
This is now updated, but in the case where we catch an unknown error I've kept the error type raised as `DvlaException` since we don't know if it's retryable or not so can't be more specific